setup.cfg

```diff
@@ -135,15 +136,14 @@ dev =
     %(testing)s
 docs =
     %(all)s
-    sphinx_autodoc_typehints==1.12.0
-    Jinja2
-    jupytext
+    sphinx<5
```

Comment:
Is that necessary if this is already in install_requires ? Or do you leave it twice because at some point it will be removed from one of the sections ?

Reply:
Maybe it isn't? I was just mirroring all the installation requirements but I honestly don't know about other workflows since I always use ".[all]". Happy to remove if it's redundant

Reply:
I think we can remove anything that's already in `install_requires`

Reply:
but `sphinx` is in `install_requires` only because of our sphinx theme has outdated dependencies https://github.com/napari/napari-sphinx-theme/issues/121 and is a temporal workaround until the theme is updated.

Our code does not require sphinx directly, but we use numpydoc which depends on the sphinx. So adding is required not to break building docs. 

It should be kept in both places to remind us that it should be fixed.

Reply:
Ok, let's leave it for the time being.